src/content/references/en/9-eurocrops.md

```diff
@@ -0,0 +1,17 @@
+---
+image: ../../../images/references/eurocrops.png
+title: EuroCrops and Random Forest
+---
+
+We trained a pipeline for field crop detection with ML on Sentinel-2 data and the EuroCrops fields. The pipeline includes all steps from retrieving the data, pre-processing, temporal aggregation over months, to the ML framework in Python. Our Python library matching the Geo Engine is used here.
+
+## Highlights
+
+- Sentinel-2 data retrieval from the AWS cloud via STAC.
+- Spatio-temporal caching of intermediate results
+- Python library to model and and execute workflows
+- Connectivity of ML framework in Python
+
+## Methodik
```

Comment:
should be Methodology

Reply:
[304b434](https://github.com/geo-engine/geoengine.de/pull/6/commits/304b4343dd4c0ea1ab9fdb71988815b9b1b5c145)